src/containers/shared/Interfaces.tsx

```diff
@@ -0,0 +1,36 @@
+export interface AccountInfoProps {
+  sequence?: number
+  ticketCount?: number
+  ownerCount?: number
+  reserve?: number
+  tick?: number
+  rate?: number
+  domain?: string
+  emailHash?: string
+  flags?: string[]
+  balance?: string
+  gravatar?: any
+  previousTxn?: string
+  previousLedger?: number
+}
```

Comment:
so these are the return values of the `rippled` APIs?

Reply:
if so the name is confusing - these are the required props for a component called `AccountInfo` but are the return values of the `account_info` request